Newsletters: add 322 (2024-09-27)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Bitcoin Stack Exchange @bitschmidty 
- [x] Merge summaries @Gustavojfe

jonatack reviewed (commented):
Looking good, an interesting newsletter

Gustavojfe commented:
just added the merge summaries, I wasn't too sure about the second one concerning custom channels... please give me feedback, thank you.

vostrnad commented on _posts/en/newsletters/2024-09-27-newsletter.md:
```diff
@@ -0,0 +1,267 @@
+  and other files are not subject to the `prune` limit and can grow in size
+  independently.
+
+- [What do I need to have set up to have `getblocktemplate` work?]({{bse}}124142)
+  User CoinZwischenzug also asks a [related question]({{bse}}124160) about how
+  to calculate the merkle root and coinbase transaction for a block. Answers to
+  both questions similarly indicate that while Bitcoin Core's `getblocktemplate`
+  can build candidate blocks of transactions and block header information, when
+  mining on non-test networks, coinbase transactions are created by mining or
+  [mining pool][topic pooled mining] software.
```
Why is the asker credited here instead of the answerers? (and I'm not asking just because that includes me)

harding replied:
Leaving this for @bitschmidty

bitschmidty replied:
> Why is the asker credited here instead of the answerers? (and I'm not asking just because that includes me)

I thought the story was easier to tell. But youre probably right to note the answeroors as well.

harding commented on _posts/en/newsletters/2024-09-27-newsletter.md:
```diff
@@ -0,0 +1,215 @@
+  cost of validating it, transferring the full history weakens the
+  privacy of previous receivers of the token.  By comparison, Shielded
+  CSV uses zero-knowledge proofs to allow verification with a fixed
+  amount of resources and without revealing previous transfers.
```
The next paragraph describes the resources required for a transfer to be _confirmed_.  This sentence describes _validation_.  From my quick skim of the paper, I'm not really clear about the actual validation costs; all I know is that the authors describe validation cost as fixed (assuming the cost to validate and scan the Bitcoin blockchain is not considered).  Leaving this hand-wave as-is due to my own ignorance.

harding commented:
> just added the merge summaries, I wasn't too sure about the second one concerning custom channels... please give me feedback, thank you.

@Gustavojfe LGTM.  I made one small change: s/Lightning Network's/LND's/ as taproot asset support is currently very specific to LND with no indication that I've seen that any other LN node will adopt support for it.

harding commented on _posts/en/newsletters/2024-09-27-newsletter.md:
```diff
@@ -0,0 +1,267 @@
+
+  The paper goes into details.  We found particularly interesting the
+  idea of trustlessly bridging bitcoin from the main blockchain to a
+  Shielded CSV (and back) without consensus changes using [BitVM][topic
```
Made a small edit that should make this clearer, but just in case it still looks wrong when you're merging, here's a quick explanation: moving BTC to a CSV thing is easy---the client is already processing Bitcoin transactions.  Moving BTC from a CSV thing back to mainnet is hard---to prove the transfer is authorized, you typically need operations not available in Script.  The paper authors claim that BitVM can work around this.

In other words, bi-directional bridging would use BitVM.

To go into unnecessary detail, I suspect there's some additional constraints that the authors haven't mentioned (or that I didn't see in my quick skim).  BitVM can verify execution of arbitrary programs---but it depends on at least one honest user independently performing verification and being able to submit a fraud proof before the money is withdrawn from the BitVM contract.  In other words, anyone with BTC in a Shielded CSV would need to come online periodically to check whether someone was trying to steal their BTC (similar to LN).

bitschmidty replied:
Thanks for clarifying. I could see how someone could have read the original "without consensus changes using BitVM" as "without consensus changes that use BitVM"

harding commented:
Made edits for all feedback (except where I replied with a comment), reviewed contributions, added lede, checked for release/RC updates (none seen), and added topic entries.  I also merged #1822 as suggested by @bitschmidty (had to move to a different PR due to some weird issue); this might cause a minor merge conflict.

Thanks everyone!

bitschmidty commented:
Made a stack exchange fixup, reviewed lede and topics (look good!), rebased on master, squashed. Thank you @harding @Gustavojfe @jonatack @vostrnad for contributing this week! 🚀
